src/app/content/practiceQuestions/components/PracticeQuestionScreen.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { routes } from '../..';
+import createTestServices from '../../../../test/createTestServices';
+import createTestStore from '../../../../test/createTestStore';
+import { book as archiveBook, shortPage } from '../../../../test/mocks/archiveLoader';
+import { mockCmsBook } from '../../../../test/mocks/osWebLoader';
+import * as Services from '../../../context/Services';
+import MessageProvider from '../../../MessageProvider';
+import { Store } from '../../../types';
+import { receiveBook } from '../../actions';
+import { LinkedArchiveTreeSection } from '../../types';
+import { formatBookData } from '../../utils';
+import { findArchiveTreeNodeById } from '../../utils/archiveTreeUtils';
+import { receivePracticeQuestionsSummary, setQuestions, setSelectedSection } from '../actions';
+import { PracticeQuestion, PracticeQuestions } from '../types';
+import { AnswerBlock } from './Answer';
+import MockQuestionData from './MockQuestionData.json';
+import Question from './Question';
+
+describe('Practice questions screen', () => {
+  let store: Store;
+  let dispatch: jest.SpyInstance;
+  let services: ReturnType<typeof createTestServices>;
+  let render: () => JSX.Element;
+  let source: LinkedArchiveTreeSection;
+
+  beforeEach(() => {
+    store = createTestStore();
+    dispatch = jest.spyOn(store, 'dispatch');
+    services = createTestServices();
+    source = findArchiveTreeNodeById(
+      formatBookData(archiveBook, mockCmsBook).tree,
+      shortPage.id
+    )! as LinkedArchiveTreeSection;
```

Comment:
we've used to use `assertDefined` instead of `!` at then end

Reply:
realized I don't need it